recipes/viennarna/meta.yaml

```
@@ -7,7 +7,7 @@ package:
 
 build:
   skip: True # [osx]
```

```suggestion
  skip: True # [osx and x86_64]
```
Right now it's skipping all osx. If you want osx-arm64 but not osx-64, you can specify it here. Otherwise, you can remove this line to build both.

---

ouch!  Thanks for pointing it out!  I'll try